service/windows/src/config.rs

```diff
@@ -154,6 +154,25 @@ fn load_config_file(conf_str: &str, conf_root: &Path) -> anyhow::Result<Config>
             );
             let baker_credentials =
                 toml_get_as!(as_str, &node, "baker_credentials").map(make_relative_path);
+            let validator_credentials =
+                toml_get_as!(as_str, &node, "validator_credentials").map(make_relative_path);
+            // In order to support legacy configuration files we support both
+            // baker_credentials and validator_credentials.
+            // We allow both, but to prevent errors we mandate that if both options are
+            // supplied they are consistent.
+            let baker_credentials = match (baker_credentials, validator_credentials) {
+                (Some(bcred), Some(vcred)) => {
+                    anyhow::ensure!(
+                        bcred == vcred,
+                        "Both `baker_credentials` and `validator_credentials` are specified, and \
+                         different."
+                    );
+                    Some(bcred)
+                }
+                (baker_credentials, validator_credentials) => {
+                    baker_credentials.or(validator_credentials)
+                }
+            };
```

Comment:
I think it would be more intuitive to ignore the `baker_credentials` if `validator_credentials` is supplied.

Reply:
I think this prevents more stupid errors. If the user supplied both and different they are doing something wrong and I think they should be told this. At least this is what I would want to be told.